recipe/meta.yaml

```
@@ -1,52 +1,83 @@
-{% set version = "1.3.0" %}
+{% set name = "segregation" %}
+{% set version = "2.5.2" %}
 
 package:
-  name: segregation
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/pysal/segregation/archive/v{{ version }}.tar.gz
-  sha256: 5a2e67c95183ef6324454bfe69a18737288df451a8ce8cccc1cec092064767e4
+  url: https://github.com/pysal/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 25b4011853c6f3562af4fdabcf507fbadee98f43da38dcd59734667c272369b7
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  skip: true  # [py<310]
 
 requirements:
   host:
-    - python >=3
+    - setuptools >=61.0
+    - setuptools-scm >=6.2
+    - python
     - pip
+    - wheel
   run:
-    - python >=3
-    - pandas
-    - geopandas
-    - matplotlib-base
-    - scikit-learn
-    - seaborn
+    - python
+    - deprecation
+    - geopandas >=0.9
+    - joblib
+    - libpysal
+    - mapclassify
+    - matplotlib
```

```suggestion
    - matplotlib-base
```
To avoid an extra size of `pyqt`